tests/foreman/cli/test_location.py

```diff
@@ -407,3 +407,27 @@ def test_negative_update_parent_with_child(self, request):
             Location.update({'id': location['id'], 'parent-id': location['id']})
         location = Location.info({'id': location['id']})
         assert location['parent'] == parent_location['name']
+
+    @pytest.mark.skip_if_open("BZ:1937009")
+    @pytest.mark.tier1
+    def test_positive_nested_location(self, request, module_org):
+        """View nested location in an organization
+
+        :id: cce4a33d-7743-47ef-b437-2539fdc73c44
+
+        :customerscenario: true
+
+        :BZ: 1937009
+
+        :expectedresults: Nested location can be viewed
+
+        :CaseImportance: Medium
+        """
+        parent_location = _location(request, {'organization-id': module_org.id})
+        location = _location(
+            request, {'parent-id': parent_location['id'], 'organization-id': module_org.id}
+        )
+        loc_list = Location.list(
+            {'search': f'title={location["title"]}', 'organization-id': module_org.id}
+        )
+        assert loc_list[0]['title'] == location['title']
```

Comment:
Assuming `location` will be placed in 1st position in the location list could fail this test case if the list return preferences changed(now by creation time/id probably) so just safely check if the id/title of location in location list.

```suggestion
        assert location['title'] in [ loc.get('title') for loc in loc_list ]
```

Reply:
@jyejare This will always work because in the previous line it searches location with the id and returns that location only so loc_list will always have one element.

Reply:
Ahh I missed that line and confused with `loc_list` variable ! Anyways you have my ACK already !